env/test

```diff
@@ -1,6 +1,7 @@
 # Add Web3 url
 NODE_URL=https://ropsten.infura.io/v3/7bfab7398ae84af3b1b70c955cfd9491
-
+# Add GAS STATION API KEY
+ETH_GAS_STATION_KEY=9c231c66f9147d1b670594483324f7203ba8a7f0e4ac2f368a4f99625373
```

Comment:
Is this a real key? If so you might need to now cancel it / make sure to remove any card details to avoid getting charged in the future @AliMeer

Reply:
@plutoegg its real and created to be checked in, all other env variables in `env/test` are present so wanted this one to be there as well. It's not linked to any payment method.